tests/test_node.py

```diff
@@ -0,0 +1,95 @@
+from PyTDF import *
+import unittest
+
```

Comment:
This file has a different indentation than the previous ones, I would make them the same